src/modules/auth/auth.service.ts

```
@@ -0,0 +1,289 @@
+  //  Création de compte
+  async registerLocal(registerDto: RegisterDto): Promise<User> {
+    const { fullName, email, password, phone } = registerDto;
+
+    const existingUser = await this.usersRepository.findOne({ where: { email } });
+    if (existingUser) {
+      throwHttpError(ErrorCode.EMAIL_ALREADY_USED, { email: email });
+    }
+
+    const hashedPassword = await bcrypt.hash(password!, 10);
+
+    const newUser = this.usersRepository.create({
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)